### PR TITLE
Feature/encrypt file

### PR DESCRIPTION
**Pull request by dmpop**
Hi Frank,

Would you mind taking a look at this? Thanks!

### CODE REVIEW

File: xml/security_cryptofs.xml [hunk 1]
```diff
@@ -245,77 +215,100 @@
...
     password automatically becomes the owner of the device.
     For devices with a file system other than FAT, change the
-    ownership explicitly for users other than &rootuser; to enable these
-    users to read or write files on the device.
-   </para><indexterm startref="idx.encrypting.remmedia" class="endofrange"/>
+    ownership explicitly for users other than &rootuser; to give them
+    the read-write access to the device.
+   </para>
+   <para>
+     If you created a virtual disk as described in <xref
```
Comment (fsundermeyer) [inline xml/security_cryptofs.xml]: If you _have_ created...

File: xml/security_cryptofs.xml [hunk 2]
```diff
@@ -245,77 +215,100 @@
...
+     linkend="sec.security.cryptofs.y2.vdisk"/> but with the loop file on a
+     removable disk, then you need to mount the file manually as follows:
+   </para>
+   <screen>sudo cryptsetup luksOpen  <replaceable>FILE</replaceable> <replaceable>NAME</replaceable>
+sudo mount /dev/mapper/<replaceable>NAME</replaceable> <replaceable>DIR</replaceable>
+   </screen>
+   <para>
+     In the commands above, the <replaceable>FILE</replaceable> refers to the
+     path to the loop file, <replaceable>NAME</replaceable> is user-definable
```
Comment (fsundermeyer) [inline xml/security_cryptofs.xml]: ... is _a_ user-definable...

File: xml/security_cryptofs.xml [hunk 3]
```diff
@@ -245,77 +215,100 @@
...
  <sect1 xml:id="sec.security.cryptofs.y2.homes">
   <title>Using Encrypted Home Directories</title>
 
   <para>
-   To protect data in home directories from being stolen and consequent
-   unauthorized access, use the &yast; user management module to enable
-   encryption of home directories. You can create encrypted home directories
-   for new or existing users. To encrypt or decrypt home directories of
-   already existing users, you need to know their login password. See
+   To protect data in home directories from an unauthorized access, use the
```
Comment (fsundermeyer) [inline xml/security_cryptofs.xml]: ...in home directories from ~~an~~ unauthorized access...

File: xml/security_cryptofs.xml [hunk 4]
```diff
@@ -370,36 +363,49 @@
...
+  <para>
+    To decrypt an encrypted file, use the following command:
+  </para>
+  <screen>gpg -d -o <replaceable>DECRYPTED_FILE</replaceable>
+  <replaceable>ENCRYPTED_FILE</replaceable></screen>
+  <para>
+    Replace <replaceable>DECRYPTED_FILE</replaceable> with the desired name for
+    the decrypted file and <replaceable>ENCRYPTED_FILE</replaceable> with the
+    encrypted file you want to decrypt.
+  </para>
```
Comment (fsundermeyer) [inline xml/security_cryptofs.xml]: You should mention, that a file encrypted this way can _only_ be decrypted with the private key of the same user that has encrypted it. To share an encrypted file with a second person, you additionally need that person's public key for encryption.

Comment (dmpop) [inline xml/security_cryptofs.xml]: Incorporated all comments, fixes, and suggestions. Thank you, Frank!

**Pull request closed without merging**